node/subsystem/src/messages.rs

```diff
@@ -285,6 +285,66 @@ impl AvailabilityStoreMessage {
 	}
 }
 
+/// A description of an error causing the chain API request to be unservable.
+#[derive(Debug, Clone)]
+pub struct ChainApiError {
```

Comment:
this could be moved to an `error` module along with `RuntimeApiError`

Reply:
We should agree on an approach how to do this - some subsystems use a per crate error type. Some use the `SubsystemResult` directly, and this adds a 3rd variant making the submodule error public.

Reply:
>  this adds a 3rd variant making the submodule error public

this was already the case with `RuntimeApiError`, I guess it needs to be public, because the requests are fallible and hence in the public API

done in d2d4891